### PR TITLE
test: bootstrap

### PR DESCRIPTION
**Pull request by joshuarli**

### CODE REVIEW

**Review by IanWoodard — CHANGES_REQUESTED** [review]
Could we stick to the same directory pattern used for the other tests or update those to match?

File: tests/test_bootstrap.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    ) as mock_proc_run:
+        coderoot = f"{tmp_path}/coderoot"
+        bootstrap.main(coderoot=coderoot, argv=("sentry",))
+        mock_add_to_known_hosts.assert_called_once()
+        mock_generate_and_configure_ssh_keypair.assert_called_once()
+        mock_brew_install.assert_called_once()
+        mock_volta_install.assert_called_once()
+        mock_direnv_install.assert_called_once()
+        mock_colima_install.assert_called_once()
+        mock_limactl_install.assert_called_once()
```
Comment (IanWoodard) [inline tests/test_bootstrap.py:47]: Nit: Could we assert that these were called with specific inputs?

**Review by IanWoodard — APPROVED** [review]
One nit, but otherwise lgtm

File: tests/test_bootstrap.py (hunk 1 above)
Comment (joshuarli) [reply tests/test_bootstrap.py:47]: they don't take any arguments

**Pull request merged**